src/schema/v1/ecommerce/types/order_party_union.ts

```diff
@@ -1,9 +1,9 @@
 import { GraphQLUnionType } from "graphql"
-import Partner from "schema/v1/partner"
+import { PartnerType } from "schema/v1/partner"
 import { UserType } from "schema/v1/user"
 
 export const OrderPartyUnionType = new GraphQLUnionType({
   name: "OrderParty",
-  types: [Partner.type, UserType],
-  resolveType: obj => (obj.__typename === "User" ? UserType : Partner.type),
+  types: [PartnerType, UserType],
+  resolveType: obj => (obj.__typename === "User" ? UserType : PartnerType),
```

Comment:
So with the new graphql-js typings, this legacy pattern of getting the type from an exported `GraphQLFieldConfig` is no longer an option. Because a field config can have any sort of graphql type (e.g. `GraphQLScalarType`), whereas some APIs such as the above need specific graphql types like `GraphQLObjectType`.

This is much better, though, and forces us to have named exports.

Reply:
yes! i was always confused about `Partner.type` vs. `PartnerType`